marketplace/api/index.ts

```diff
@@ -7,16 +7,15 @@ app.get('/', (req, res) => {
   res.send('Dagster Marketplace API');
 });
 
-//
-// Serve `__generated__` files (e.g. `index.json` at `/api/integrations/index.json`)
-//
-app.use('/api/integrations', express.static(path.join(__dirname, '../__generated__')));
+/*
+ * Serve `__json__` files (e.g. `index.json` at `/api/integrations/index.json`)
+ */
+app.use('/api/integrations', express.static(path.join(__dirname, '../__json__')));
 
-// Servce static assets, for example:
-//
-// public/images/dagster-primary-horizontal.svg -> /assets/images/dagster-primary-horizontal.svg
-//
-app.use('/assets', express.static(path.join(__dirname, '../public')));
+/*
+ * Serve `__logos__` files (e.g. `dagster.svg` at `/api/logos/dagster.svg`)
+ */
+app.use('/api/logos', express.static(path.join(__dirname, '../__logos__')));
```

Comment:
Might be a little weird to have images served at an `/api` endpoint, I haven't seen that before. But I'm OK with it.

Reply:
This is mostly just me being lazy about updating the `vercel.json` to accommodate CORS on these, haha. I agree though, so I'll go ahead and clean it up.